zingolib/src/wallet/transaction_records_by_id/trait_inputsource.rs

```diff
@@ -126,31 +124,40 @@ impl InputSource for TransactionRecordsById {
         anchor_height: zcash_primitives::consensus::BlockHeight,
         exclude: &[Self::NoteRef],
     ) -> Result<SpendableNotes<Self::NoteRef>, Self::Error> {
-        let mut unselected =
-            self.get_spendable_note_ids_and_values(sources, anchor_height, exclude);
-
+        let mut unselected = self
+            .get_spendable_note_ids_and_values(sources, anchor_height, exclude)
+            .into_iter()
+            .map(|(id, value)| NonNegativeAmount::from_u64(value).map(|value| (id, value)))
```

Comment:
This replaces `u64` with `Zatoshi` unless there's an error.  How about:

`// swap u64s for Zats`

Reply:
sure